packages/angular/migrations.json

```diff
@@ -1305,6 +1305,15 @@
           "alwaysAddToPackageJson": false
         }
       }
+    },
+    "16.8.0": {
+      "version": "16.8.0-beta.0",
```

Comment:
Maybe check with @FrozenPandaz if this is intended to be released with the next minor or the next patch version.